static/js/components/LearnerSearch.js

```diff
@@ -128,6 +129,16 @@ export default class LearnerSearch extends SearchkitComponent {
       {openLearnerEmailComposer: this.props.openLearnerEmailComposer},
       LearnerResult
     );
+    this.forceSearch = true;
+  }
+
+  loaded = (): boolean => {
+    if (this.forceSearch) {
```

Comment:
I notice that when the components first render then the hit search url, 
Without rendering for first time search wont initiate.

isInitialLoading() method from searchkit get update when search is complete for first time